module-utils/utility/Version.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2017-2024, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include "Version.hpp"
+
+#include <service-db/agents/settings/SystemSettings.hpp>
+#include <service-desktop/Constants.hpp>
```

Comment:
is this necessary?

Reply:
`#include <service-db/agents/settings/SystemSettings.hpp>` is needed by `settings::SystemProperties::osCurrentVersion`
but `#include <service-desktop/Constants.hpp>` isn't necessary here. Thanks ;)

Reply:
It's not important now, I removed those includes.